.gitignore

```diff
@@ -3,3 +3,4 @@ demo/helper-scripts/*.pdf
 demo/helper-scripts/*.log
 /kustomization.yaml
 coverage.out
+vendor/
```

Comment:
I think it's better not to have this in gitignore as we don't ship with the vendored deps. With this change an out-of-date vendor/ directory get's ignored and easily causes unexpected build results

Reply:
ok